sources/packages/backend/apps/workers/src/controllers/cra-integration/cra-integration.dto.ts

```diff
@@ -1,18 +1,27 @@
+import { APPLICATION_ID } from "@sims/services/workflow/variables/assessment-gateway";
+import {
+  INCOME_VERIFICATION_COMPLETED,
+  INCOME_VERIFICATION_ID,
+  REPORTED_INCOME,
+  SUPPORTING_USER_ID,
+  TAX_YEAR,
+} from "@sims/services/workflow/variables/cra-integration-income-verification";
+
 export interface CreateIncomeRequestJobInDTO {
-  applicationId: number;
-  taxYear: number;
-  reportedIncome: number;
-  supportingUserId?: number;
+  [APPLICATION_ID]: number;
```

Comment:
for my info, how does this work I mean the `​[x] : number​​`

Reply:
This is equivalent to having the property declared as `applicationId: number;` where "applicationId" is the content of the const `APPLICATION_ID`.
"applicationId" is a variable declared on Camunda, if it is changed we can change it in one place only in workers and see all the affected points easily.

Reply:
The property name is variable. Then are we going to use [APPLICATION_ID] anywhere in the code to refer to that variable?

Reply:
No need, typescript handles it pretty well. In the code we just use `object.applicationId`.